.travis.yml

```diff
@@ -16,6 +16,7 @@ before_install:
     - travis_retry conda create --yes -n test python=$PYTHON --file requirements.txt
     - source activate test
     - conda install --yes --file requirements-dev.txt;
+    - pip install git+https://github.com/python-visualization/branca.git
```

Comment:
@ocefpaf 
> However, let's hold the merge until we have a first release of branca. When projects are tied together like this the first dependency dictates the releases.

This line downloads and install the master of branca, meaning folium tests will run with branca's present master.
I guess this is too dangerous because it breaks tests reproducibility. But I know it's possible to ask pip for a specific commit, so that we could live (on folium's master) on a version of branca that's not necessarily released.

Depending on your reply, I'll either freeze a branca commit into this line, or create a `v0.0.0` of branca and ask for it in this line.